package.json

```diff
@@ -82,6 +82,7 @@
         "react-tagsinput": "^3.19.0",
         "react-time-ago": "^5.0.4",
         "react-toastify": "^5.0.0",
+        "react-tooltip": "^3.10.0",
```

Comment:
Great work @Peace-Apple, how is this package helpful.

Reply:
That package enables us to include a text that will appear whenever a user hovers over that bookmark icon, in this case I display a text bookmark or unbookmark accordingly, you could check out the deployment to see how that works